conf/application.conf

```diff
@@ -194,6 +194,7 @@ feedback {
 }
 
 accessibility-link-url = "/cds-file-upload-service/help/accessibility"
+accessibility-statement.service-path = "/cds-file-upload-service/help/accessibility"
```

Comment:
IMHO it would be better to keep urls in `urls` section

Reply:
No it has to be there.